### PR TITLE
Added virtual caps doc

### PR DESCRIPTION
**Pull request by tonyheadford**
### Context
[Trello card](https://trello.com/c/OOGyg2Oy/1378-add-virtual-cap-doc)
Add some documentation to cover virtual caps. Hopefully this will aid understanding especially for the next team.

### Changes proposed in this pull request
Add virtual cap pools doc.
Merge support doc about adding a school to a virtual cap pool into virtual caps doc.

### Guidance to review
Being quite close to this, I may have skipped over or missed some things that may not be obvious so I am expecting to need to tweak this.

### CODE REVIEW

File: docs/virtual_cap_pools.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+# Virtual Cap Pools
+
+## Overview
+
+Before virtual cap pools were implemented, each school would have a `SchoolDeviceAllocation` for devices and possibly another `SchoolDeviceAllocation` for routers.  The `SchoolDeviceAllocation` contains the `allocation`, `cap` (the amount of their allocation they can currently order) and a `devices_ordered` value indicating how much of the `cap` had been used.
+
+The `cap` values are sent automatically to Computacenter so that they are able to control the amount available for the school to order via Techsource.  The `devices_ordered` values are updated by Computacenter once a school has placed an order.
+
+For a responsible body with many centrally managed schools, there became a need to be able to control where devices were delivered in order to facilitate centralised IT management for the organisation.
```
Comment (aldavidson) [inline docs/virtual_cap_pools.md:9]: I think we could maybe do with a little bit more on this point - what problems did it cause, for whom?

**Review by aldavidson — APPROVED** [review]
I think this is great - thanks! 
Love the diagrams. There's only one bit that I think could do with expanding on a little bit, comment below (not a blocker though)

File: docs/virtual_cap_pools.md (hunk 1 above)
Comment (tonyheadford) [reply docs/virtual_cap_pools.md]: Thanks, I've reworked this bit.

**Review by aldavidson — APPROVED** [review]
Yep, that's all it needed, thanks :+1:

**Pull request merged**